nixos/modules/security/acme.nix

```diff
@@ -297,12 +297,7 @@ in
                   wantedBy = mkIf (!config.boot.isContainer) [ "multi-user.target" ];
                   serviceConfig = {
                     Type = "oneshot";
-                    # With RemainAfterExit the service is considered active even
-                    # after the main process having exited, which means when it
-                    # gets changed, the activation phase restarts it, meaning
-                    # the permissions of the StateDirectory get adjusted
-                    # according to the specified group
-                    RemainAfterExit = true;
+                    SuccessExitStatus = [ "0" "1" ];
```

Comment:
What's this for?

Reply:
Maybe an artifact of reverting. The `SuccessExitStatus` thing was removed in 1e3607d331a650e958b48e0c6a9231e68dd023f8

Reply:
Thanks for the catch